README.md

```diff
@@ -117,60 +117,61 @@ The following code creates an element with the specified `tag`
 v(tag: string): HNode[];
 ```
 
-where `tag` is in the form: element.className(s)#id, e.g.
```

Comment:
We don't use it in any of our widgets and it would not be considered a "best practice" to. So I think it's best to not remove it from the readme.